dist/minimal-lightbox.js

```diff
@@ -1215,7 +1216,7 @@ var Zoomable = (function () {
         }
       } else {
         var translate = getTranslate(element);
-        var scale = getZoom(element);
+        var scale = getZoom(element, this.useActualMax);
```

Comment:
getZoom() being outside the Class makes it odd when we add options like here.
It would be better to have the function inside the Class so wouldn't even bother to pass the element.